src/fen_util.rs

```diff
@@ -0,0 +1,33 @@
+use crate::board::Chessboard;
+
+#[rustfmt::skip]
+pub fn place_pieces(chessboard: &mut Chessboard, fen_rows: &str) {
```

Comment:
Perhaps we should have this return a result so we can catch the errors downstream?
Ideally it would never error because we would check for a valid fen first, but what's ideal almost never happens.

Reply:
Yah I need to think more on what we should return.

Reply:
Issue created.